third-party/realsense-file/rosbag/roscpp_serialization/include/ros/serialization.h

```diff
@@ -369,7 +363,7 @@ struct VectorSerializer
  * \brief Vector serializer, specialized for non-fixed-size, non-simple types
  */
 template<typename T, class ContainerAllocator>
-struct VectorSerializer<T, ContainerAllocator, typename boost::disable_if<mt::IsFixedSize<T> >::type >
+struct VectorSerializer<T, ContainerAllocator, typename std::enable_if< !mt::IsFixedSize<T>::value >::type >
```

Comment:
They used `disable_if`, what does this change means?

Reply:
enable_if( ! ... ) == disable_if( ... )